bitnami/mariadb/README.md

```diff
@@ -189,6 +190,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `secondary.affinity`                           | Affinity for MariaDB secondary pods assignment                                                                        | `{}`            |
 | `secondary.nodeSelector`                       | Node labels for MariaDB secondary pods assignment                                                                     | `{}`            |
 | `secondary.tolerations`                        | Tolerations for MariaDB secondary pods assignment                                                                     | `[]`            |
+| `secondary.topologySpreadConstraints`          | Topology Spread Constraints for MariaDB secondary pods assignment                                                     | `[]`            |
```

Comment:
```suggestion
| `secondary.topologySpreadConstraints`          | Topology Spread Constraints for MariaDB secondary pods assignment                                                     | `{}`            |
```
It should be an object instead of an array